packages/validator-bonds-cli-institutional/README.md

```diff
@@ -232,7 +229,7 @@ The resulting APY is then weighted by the ratio of the Select stake
 The APY is calculated using the formula:
 
 ```typescript
-APY = (1 + rewardsPerEpoch / stakedAmount) ^ epochsInYear - 1
+APY = (1 + rewardsPerEpoch / stakedAmount) ^ (epochsInYear - 1)
```

Comment:
```suggestion
APY = ((1 + rewardsPerEpoch / stakedAmount) ^ epochsInYear) - 1
```
Not sure if this was edited by the linter, but the parentheses are actually wrong. If they should really be there (not necessary), then as I'm suggesting.